prime-router/docs/observability/common-queries.md

```diff
@@ -0,0 +1,64 @@
+# Common Queries in KQL
+
+## General tips
+- Use the `Time range` selector in the Azure query dialog to specify what time range
+you want your results from.
+- The value of `null` for a receiver in the `ReportRouteEvent` means that report was not routed to any receiver
```

Comment:
I'm thinking this doc should just be a section in the azure-events.md doc. Common Queries in KQL can be a section in that doc with everything in this doc as is except for this line -- we should have another section in the main doc titled something like "ReportStream Events Glossary" with a short description of each event and how it is meant to be used and where it is called in the pipeline. Then, you can put this specific line I am commenting on as a blurb in the "ReportRouteEvent" section?

Reply:
Makes sense to me 👍